testing/benchmark/Makefile

```diff
@@ -54,17 +48,8 @@ REPO_ROOT:=$(abspath ${MAKEFILE_DIR}/../../)
 include ${MAKEFILE_DIR}/../../go.mk
 
 .PHONY: auth
-auth: init-aws-profile
-	@okta-awscli --profile $(AWS_PROFILE)
-
-.PHONY: init-aws-profile
-init-aws-profile: ~/.aws/credentials
-	@grep $(AWS_PROFILE) ~/.aws/credentials > /dev/null || \
-		echo "[$(AWS_PROFILE)]\naws_access_key_id = x\naws_secret_access_key = x" >> ~/.aws/credentials
-
-~/.aws/credentials:
-	@mkdir -p ~/.aws
-	@touch $@
+auth:
+	@okta-aws-cli --profile $(AWS_PROFILE) --write-aws-credentials
```

Comment:
We need to use the `elastic-okta-awscli` - the only CLI available for some Elastic employees. The current `elastic-observability` should work with that tool.

We can sync up and discuss this if needed

Reply:
Updated to use `--short-user-agent`